### PR TITLE
[SPARK-46438][UI] Fix the broken scope of toggleFramegraph when registering onclick event for the Frame Graph header

### PR DESCRIPTION
**Pull request by yaooqinn**
### What changes were proposed in this pull request?

This PR fixes the broken scope of toggleFramegraph when registering onclick event for the Frame Graph header. Otherwise, the Frame Graph cannot be  collapsed.

### Why are the changes needed?

bugfix for UI.

### Does this PR introduce _any_ user-facing change?

no
### How was this patch tested?

added a new test for toggleFramegraph in ui-tests.
### Was this patch authored or co-authored using generative AI tooling?

no

### CODE REVIEW

**Comment by yaooqinn** [comment]
cc @sarutak @dongjoon-hyun @HyukjinKwon

File: ui-test/tests/flamegraph.test.js [hunk 1]
```diff
@@ -38,3 +40,20 @@ test('drawFlamegraph', function () {
   expect($("[name=apache] title").html()).toBe('apache (100.000%, 2 samples)');
   expect($("[name=spark] title").html()).toBe('spark (50.000%, 1 samples)');
 });
+
+test('toggleFlamegraph', function () {
```
Comment (dongjoon-hyun) [inline ui-test/tests/flamegraph.test.js:44]: Thank you for adding this, @yaooqinn .

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.
Merged to master.

**Comment by yaooqinn** [comment]
Thank you @dongjoon-hyun

**Pull request closed without merging**